vendor/k8s.io/kubernetes/pkg/api/validation/validation.go

```diff
@@ -2544,6 +2544,17 @@ func ValidateLimitRange(limitRange *api.LimitRange) field.ErrorList {
 			}
 		}
 
+		if limit.Type == api.LimitTypePersistentVolumeClaim {
+			_, minQuantityFound := limit.Min[api.ResourceStorage]
```

Comment:
@markturansky - i must have missed this in the review upstream.  i should be able to set just a min and not a max, or a max and not a min, disagree?